torch/_lazy/computation.py

```diff
@@ -0,0 +1,15 @@
+import torch._C._lazy
+import torch._C._lazy_ts_backend
+
+def get_tensors_ts_device_data_node(tensors):
+    """Return tensor ids and eager tensors for DeviceData nodes in the
+       IR for the passed in lazy tensors."""
+    return torch._C._lazy_ts_backend._get_tensors_ts_device_data_node(tensors)
+
+def get_graph_hash(tensors):
+    """Return the graph hash for the passed in lazy tensors"""
+    return torch._C._lazy._get_graph_hash(tensors)
+
+def run_cached_graph(hash_str, graph_inputs):
```

Comment:
maybe for now you can leave this 'run_cached_graph' api under the ts_backend namespace, and later we can consider a generic api at this level if we can make it work for all backends